File: TradingEAC-Java-PriceService/src/main/java/tradingeac/java/priceservice/models/Quote.java
package tradingeac.java.priceservice.models;

import lombok.Getter;
import lombok.Setter;

public class Quote {
    private String symbol;
    public void setSymbol(String value){
        symbol = value;
    }
    public String getSymbol(){
        return symbol;
    }

    private double c;
    public void setC(double value){
        c = value;
    }
    public double getC(){
        return c;
    }

    private double h;
    public void setH(double value){
        h = value;
    }
    public double getH(){
        return h;
    }

    private double i;
    public void setI(double value){
        i = value;
    }
    public double getI(){
        return i;
    }

    private double o;
    public void setO(double value){
        o = value;
    }
    public double getO(){
        return o;
    }

    private double pc;
    public void setPC(double value){
        pc = value;
    }
    public double getPC(){
        return pc;
    }

    private long t;
    public void setT(long value){
        t = value;
    }
    public long getT(){
        return t;
    }

}

File: TradingEAC-DotNet-PriceService/Models/Quote.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TradingEAC.DotNet.PriceService.Models
{
    public class Quote
    {
        public string symbol { get; set; }
        public decimal c { get; set; }
        public decimal h { get; set; }
        public decimal i { get; set; }
        public decimal o { get; set; }
        public decimal pc { get; set; }
        public long t { get; set; }        
    }
}
